pkg/controller/operators/olm/overrides/config.go

```diff
@@ -29,13 +29,20 @@ func (o *operatorConfig) GetConfigOverrides(ownerCSV ownerutil.Owner) (envVarOve
 		return
 	}
 
+	if owner.Spec.Config == nil {
+		return
+	}
+
```

Comment:
There are a number of unrelated changes coming in from the `github.com/operator-framework/api` version bump that can be done in a separate PR.
E.g see: https://github.com/operator-framework/api/pull/97

Reply:
Seems like https://github.com/operator-framework/operator-lifecycle-manager/pull/2010 already does this so I'll rebase.